search/result.hpp

```diff
@@ -65,6 +65,10 @@ class Result
     // Valid for any result.
     osm::YesNoUnknown m_isOpenNow = osm::Unknown;
 
+    int m_minutesUntilOpen = 0;
+
+    int m_minutesUntilClosed = 0;
```

Comment:
```suggestion
    int m_minutesUntilOpen = 0;
    int m_minutesUntilClosed = 0;
```

Is int the best type to store these values? Can the value be negative? Can it be uint16_t?

Reply:
Changed to uint16_t